### ISSUE TITLE
Upload of PDF document(s) -- Front End

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#1]
## User story
1. As a user
2. I need to upload PDF documents
3. So that documents can be stored in the application

## Acceptance criteria
* User can either drag and drop files or select files to upload.
* If the document is of required .pdf format, upload and store the document.
* If the document is not of the required .pdf format, return an error message stating wrong format and upload cannot be done.
* User should be notified when document(s) are uploaded successfully.
* User should be notified if something went wrong in the upload process.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

Document upload page:
![Image](https://github.com/amosproj/amos2024ss05-knowledge-graph-extractor/assets/92937378/811e0bf3-5855-4ffd-98ac-555b67d4df38)

Load document:
![Image](https://github.com/amosproj/amos2024ss05-knowledge-graph-extractor/assets/92937378/660b4713-b7d6-4801-a6ba-1b63f45bf25f)

Succesful:
![Image](https://github.com/amosproj/amos2024ss05-knowledge-graph-extractor/assets/92937378/470f0751-240a-4b5f-b0a7-59114a157b1f)

Error:
![Image](https://github.com/amosproj/amos2024ss05-knowledge-graph-extractor/assets/92937378/6ad0e1a4-3a68-4d60-ab56-ddebe1435e32)

Loaded documents listed:
![Image](https://github.com/amosproj/amos2024ss05-knowledge-graph-extractor/assets/92937378/aeb4c69c-f77a-43f1-a4fa-c3c29d764db3)
